jwql/tests/test_api_views.py

```diff
@@ -73,6 +74,7 @@
     urls.append('api/{}/thumbnails/'.format(rootname))  # thumbnails_by_rootname
 
 
+@skipIf(ON_GITHUB_ACTIONS, "Can't access webpage without VPN access")
```

Comment:
Could this be changed to use `@pytest.mark.skipif()` instead of `unittest.skipIf()`? Just so that we are not adding another dependency if we don't have to?